packages/blocks/src/api/index.native.js

```diff
@@ -2,6 +2,8 @@ export {
 	cloneBlock,
 	createBlock,
 	switchToBlockType,
+	getBlockTransforms,
+	findTransform,
```

Comment:
There doesn't seem to be that many differences in this file anymore. Have you tried removing the native variant?